tests/lib/prepare.sh

```diff
@@ -345,6 +345,34 @@ repack_snapd_snap_with_deb_content() {
     rm -rf "$UNPACK_DIR"
 }
 
+repack_core20_snap_with_tweaks() {
+    local CORE20SNAP="$1"
+    local TARGET="$2"
+
+    local UNPACK_DIR="/tmp/core20-unpack"
+    unsquashfs -no-progress -d "$UNPACK_DIR" "$CORE20SNAP"
+
+    mkdir -p "$UNPACK_DIR"/etc/systemd/journald.conf.d
+    cat <<EOF > "$UNPACK_DIR"/etc/systemd/journald.conf.d/to-console.conf
+[Journal]
+ForwardToConsole=yes
+TTYPath=/dev/ttyS0
+MaxLevelConsole=debug
+EOF
+    mkdir -p "$UNPACK_DIR"/etc/systemd/system/snapd.service.d
+cat <<EOF > "$UNPACK_DIR"/etc/systemd/system/snapd.service.d/logging.conf
+[Service]
+Environment=SNAPD_DEBUG_HTTP=7 SNAPD_DEBUG=1 SNAPPY_TESTING=1 SNAPD_CONFIGURE_HOOK_TIMEOUT=30s
+StandardOutput=journal+console
+StandardError=journal+console
```

Comment:
should we maybe just set this always for snapd on ubuntu core? seems like a very useful thing to always have snapd logs on the console no matter what, and we shouldn't be logging anything by default which is sensitive afaik